client/app/views/signup.html

```diff
@@ -1,12 +1,14 @@
 <div class="container"  style="opacity:0.85">
   <div class="card-panel hoverable">
-    <form ng-submit='signup()'>
+    <form ng-submit='advanceToFoodPrefs(user)'>
       <input type="text" class="" placeholder="username" ng-model="user.username">
-      <input type="email" class="" placeholder="email" ng-model="user.email">
-      <input type="email" class="" placeholder="confirm email" ng-model="user.confirmemail">
+      <input type="text" class="" placeholder="firstname" ng-model="user.firstname">
+      <input type="text" class="" placeholder="lastname" ng-model="user.lastname">
+      <input type="text" class="" placeholder="location" ng-model="user.location">
+      <input type="text" class="" placeholder="email" ng-model="user.email">
       <input type="password" class="" placeholder="password" ng-model="user.password">
-      <input type="password" class="" placeholder="confirm password" ng-model="user.confirmpassword">
-      <input type="submit" class="" value="Sign Up"><h6 class="right right-align">Already a member? <a ui-sref="main.signin">Click here </a>to sign in</h6>
+      <input type="password" class="" placeholder="confirm password">
+      <input type="submit" class="" value="Continue" ng-click=""><h6 class="right right-align" style="opacity:1">Already a member? <a ui-sref="main.signin">Click here </a>to sign in</h6>
```

Comment:
I don't think this needs an ng-click because the submission is happening in the ng-submit on the form element.